src/Lykke.Quintessence.Job/PeriodicalHandlers/DefaultBalanceCheckScheduler.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using Autofac;
+using Common;
+using JetBrains.Annotations;
+using Lykke.Common.Log;
+using Lykke.Quintessence.Domain.PeriodicalHandlers;
+using Lykke.Quintessence.Domain.Services;
+
+namespace Lykke.Quintessence.PeriodicalHandlers
+{
+    [UsedImplicitly]
+    public class DefaultBalanceCheckScheduler : IBalanceCheckScheduler, IStartable, IStopable
+    {
+        private readonly TimerTrigger _timerTrigger;
+        
+        public DefaultBalanceCheckScheduler(
+            IBalanceCheckSchedulingService balanceCheckSchedulingService,
+            ILogFactory logFactory)
+        {
+            _timerTrigger = new TimerTrigger
+            (
+                nameof(DefaultBalanceCheckScheduler),
+                TimeSpan.FromMinutes(15),
```

Comment:
Maybe it's better to get value from settings.

Reply:
Opened an issue